api/lib/source/polkadotV0-source.js

```diff
@@ -24,22 +29,61 @@ class polkadotAPI {
     this.db = db
   }
 
+  initialize(config) {
+    this.context = config.context
+    // manually set cache to checking it
+    this.cache = new InMemoryLRUCache()
+    this.httpCache = new HTTPCache(this.cache, this.httpFetch)
+  }
+
   setReducers() {
     this.reducers = require('../reducers/polkadotV0-reducers')
   }
 
+  async getRetry(url, intent = 0) {
```

Comment:
this is the same code as in Cosmos. let's refactor

Reply:
I tried with no luck, you need access to `RESTDataSource` methods in the function and that is not possible outside the source class.

Reply:
done and pushed. can you check?